enable-walletconnect.patch

```diff
@@ -1,60 +1,199 @@
+-   INVALID_PAYLOAD: 5003,
+- };
+- 
+--function isWalletConnectEnabled() {
+--  return false;
+--  /*
+-+function* isWalletConnectEnabled() {
+-   const walletConnectEnabled = yield call(checkForFeatureFlag, WALLET_CONNECT_FEATURE_TOGGLE);
+- 
+-   return walletConnectEnabled;
+--  */
+- }
+- 
+- function* init() {
+diff --git a/ios/Podfile.lock b/ios/Podfile.lock
+index 676a0a0..28ffa36 100644
+--- a/ios/Podfile.lock
++++ b/ios/Podfile.lock
+@@ -389,6 +389,13 @@ PODS:
+   - React-jsinspector (0.72.5)
+   - React-logger (0.72.5):
+     - glog
++  - react-native-compat (2.12.2):
++    - RCT-Folly (= 2021.07.22.00)
++    - React-Core
++  - react-native-get-random-values (1.11.0):
++    - React-Core
++  - react-native-netinfo (11.3.1):
++    - React-Core
+   - react-native-pager-view (5.4.25):
+     - React-Core
+   - react-native-randombytes (3.6.1):
+@@ -612,6 +619,9 @@ DEPENDENCIES:
+   - React-jsiexecutor (from `../node_modules/react-native/ReactCommon/jsiexecutor`)
+   - React-jsinspector (from `../node_modules/react-native/ReactCommon/jsinspector`)
+   - React-logger (from `../node_modules/react-native/ReactCommon/logger`)
++  - "react-native-compat (from `../node_modules/@walletconnect/react-native-compat`)"
++  - react-native-get-random-values (from `../node_modules/react-native-get-random-values`)
++  - "react-native-netinfo (from `../node_modules/@react-native-community/netinfo`)"
+   - react-native-pager-view (from `../node_modules/react-native-pager-view`)
+   - react-native-randombytes (from `../node_modules/react-native-randombytes`)
+   - react-native-safe-area-context (from `../node_modules/react-native-safe-area-context`)
+@@ -717,6 +727,12 @@ EXTERNAL SOURCES:
+     :path: "../node_modules/react-native/ReactCommon/jsinspector"
+   React-logger:
+     :path: "../node_modules/react-native/ReactCommon/logger"
++  react-native-compat:
++    :path: "../node_modules/@walletconnect/react-native-compat"
++  react-native-get-random-values:
++    :path: "../node_modules/react-native-get-random-values"
++  react-native-netinfo:
++    :path: "../node_modules/@react-native-community/netinfo"
+   react-native-pager-view:
+     :path: "../node_modules/react-native-pager-view"
+   react-native-randombytes:
+@@ -830,6 +846,9 @@ SPEC CHECKSUMS:
+   React-jsiexecutor: ff70a72027dea5cc7d71cfcc6fad7f599f63987a
+   React-jsinspector: aef73cbd43b70675f572214d10fa438c89bf11ba
+   React-logger: 2e4aee3e11b3ec4fa6cfd8004610bbb3b8d6cca4
++  react-native-compat: ea766c4bcad72b3cb36765a97a846c9551b5b802
++  react-native-get-random-values: 21325b2244dfa6b58878f51f9aa42821e7ba3d06
++  react-native-netinfo: bdb108d340cdb41875c9ced535977cac6d2ff321
+   react-native-pager-view: da490aa1f902c9a5aeecf0909cc975ad0e92e53e
+   react-native-randombytes: 421f1c7d48c0af8dbcd471b0324393ebf8fe7846
+   react-native-safe-area-context: 68b07eabfb0d14547d36f6929c0e98d818064f02
 diff --git a/shim.js b/shim.js
-index 73c130d..10d894b 100644
+index 73c130d..deae975 100644
```

Comment:
Why do you have the `shim` file twice here?

Reply:
Fixed, can you check again?